### ISSUE TITLE
Nextcloud guide on Landchad still broken in July 2022

### ISSUE THREAD

**Issue by Rudy-MS** [LukeSmithxyz/landchad#208]
Hello there.

I apologize for reopening this issue. I could not think of anything else to do. Tried emailing Matthew Evan(the author for guide) but his email address(nor his website(https://matthewevan.xyz/) does not work at this moment in time.

As far as I can tell this guide still does not work even though it got closed. Was thinking of emailing Luke but I'm sure he gets slammed with other emails...hoping a friendly internet stranger will help us out. 

Tried the guide several time and I still cannot get it to work. Even tried what the previous post suggested to no avail. For more info please see issue that was closed below. 

If anyone knows how to contact the author(Matthew) please let him know that people are trying to use his guide and are having issues.

"Would like to say that I am also experiencing this same issue. 

TLDR: Followed https://landchad.net/nextcloud.html instructions.....twice.....no joy just get 404 error using "example.org/nextcloud" 

Interestingly enough, if I delete the "mywebsite" html link in the "/etc/nginx/sites-enabled" directory and try "example.org/nextcloud" I am greeted by Nextcloud setup/logon page....aka nextcloud seems to work if it is the only link in the said directory. 

As soon as I create a new "mywebsite" link following "https://landchad.net/nginx.html" it brings me back to the 404 page for "example.org/nextcloud" 

Makes me think that the html page "nextcloud" somehow needs adjusting to work with the(or any other html page) "mywebsite" page. 

I am hoping someone has an idea what to do with the "nextcloud" page or perhaps the author will respond to our errors/woes. 

Apologies if I have missed anything. I am a newb to Nextcloud and self-hosting."

_Originally posted by @Rudy-MS in https://github.com/LukeSmithxyz/landchad/issues/151#issuecomment-1133790386_

**Comment by LukeSmithxyz** [comment]
This and a couple other tutorials I plan on installing this week for bugtesting. I'll be able to push fixes when I can, but anyone familiar with Nextcloud can chime in the meantime. I'm going to guess this is an issue with the Nginx config file.

**Comment by LukeSmithxyz** [comment]
So I think your issue is that Certbot does you dirty and adds Nextcloud stuff to the default Nginx config file. Same thing happened to me. You'll want to go to the `mywebsite` config file and remove all the stuff related to your Nextcloud domain.

I at long last tried this out and got it working. I'll publish a revision soon.

**Comment by MattMadness** [comment]
I would like to apologize. I self host and am in the process of a move, only recently did I get a few services back up excluding email and a few others thanks to a friend who also self hosts. Therefore I was not able to recieve your email.

I believe that it is in our best interest to move over to a subdomain. At the time I first wrote this article I used a subdirectory, and I thought it was the better solution. More benefit will come out of using a subdomain as I have found, especially with ActivityPub federation. 

Sorry for any inconvinences.

**Comment by Rudy-MS** [comment]
Apologizes for the late reply. I am self employed and my business takes most of my down time. 

I finally was able to try out the new guide and it worked perfectly! 

Thank you Luke for taking the time to revise it and try it out yourself. Much appreciated! 

No worries Matt, I appreciate the concern and your work. 

God Bless!
